src/Symfony/Bundle/FrameworkBundle/Resources/config/debug_prod.php

```diff
@@ -25,7 +25,6 @@
                 null, // Log levels map for enabled error levels
                 param('debug.error_handler.throw_at'),
                 param('kernel.debug'),
-                service('debug.file_link_formatter'),
```

Comment:
This means that we just broke cross-compat between 5.4 & 6.0. composer.json needs to be updated.
On 5.4, what about still allowing v6 by doing a reflection check?

Reply:
see https://github.com/symfony/symfony/pull/42235